### ISSUE TITLE
[REQUEST] Save the position and size of Pop out windows

### ISSUE THREAD

**Issue by Finn-D-AEWG** [flybywiresim/aircraft#1371]
**Is your feature request related to a problem? Please describe.**
yes, because if I open the pop out windows with Alt Gr and then restart the flight, these windows are gone

**Describe the feature you'd like to see implemented**
 That the pop-out windows (click displays with Alt Gr) are in the same position and size when I start a new flight with the A32NX. It should therefore be possible to save the position and size of the pop-out windows.

**References**
 This feature is only for simulators...:)

**Additional context**
 This would be very important for me, as I am currently building an A320 home cockpit and don't want to drag every display from the A320NX into my own monitor before every flight and then want to adjust the size. Thanks in advance:)

Finn_A320#3595

**Comment by lukecologne** [comment]
I don't think we are able to implement that feature. This is something that Asobo needs to implement. You should open a Zendesk report so they know this is something you want.

**Comment by Pilot4XP** [comment]
> 
> 
> I don't think we are able to implement that feature. This is something that Asobo needs to implement. You should open a Zendesk report so they know this is something you want.

I've already done this 2 times and in 2 months still nothing has happened.

**Comment by Pilot4XP** [comment]
> 
> 
> > This would be very important for me, as I am currently building an A320 home cockpit and don't want to drag every display from the A320NX into my own monitor before every flight and then want to adjust the size. Thanks in advance:)
> 
> A solution would be to write a script that automatically moves them.

I thought something like that too. Maybe someone can do this, I would be very happy

**Comment by Pilot4XP** [comment]
Any update?

**Comment by tracernz** [comment]
This would need to be raised with Microsoft on the sim side. There's nothing we can do to control it.
